docs/source/tutor/_hello_world.ipynb

```diff
@@ -0,0 +1,698 @@
+    "- `RE.subscribe(BestEffortCallback())` sets up a subscription to the RunEngine instance (`RE`) that will automatically print a summary of the run data at the end of each run.\n",
+    "</details>"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can now load the `quick_hello.py` file located in the user directory and execute its contents in the current Python environment."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Loading 'Hello, World!' example.\n"
+     ]
+    }
+   ],
+   "source": [
+    "%run -i ~/bluesky/user/quick_hello.py"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Note**: The `Loading 'Hello, World!' example.` text came from a `print()` statement in the file.\n",
+    "\n",
+    "This also loads `hello_world()`, a demonstration bluesky *plan*.  We can run the `hello_world()` function in the `RE` RunEngine instance, which will execute the corresponding experimental plan. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "\n",
+      "Transient Scan ID: 1     Time: 2023-04-21 07:46:14\n",
+      "Persistent Unique Scan ID: 'ac400921-948c-453a-814d-f8720ba7b783'\n",
+      "New stream: 'primary'\n",
+      "+-----------+------------+------------+\n",
+      "|   seq_num |       time |      hello |\n",
+      "+-----------+------------+------------+\n",
+      "|         1 | 07:46:14.4 |          1 |\n",
+      "+-----------+------------+------------+\n",
+      "generator count ['ac400921'] (scan num: 1)\n",
+      "\n",
+      "\n",
+      "\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "('ac400921-948c-453a-814d-f8720ba7b783',)"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "RE(hello_world())"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can retrieve data for **the most recent run** stored in the `cat` databroker instance, and assign it to the a variable `run`. The data is available until we quit the IPython session; to access data later, see <a ref=https://bcda-aps.github.io/bluesky_training/howto/_after_measurement.html> here </a>."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run = cat[-1]\n",
```

Comment:
term(s) | `databroker.catalog` version | description
--- | --- | ---
`db` | v1 | original `Broker`-style databroker catalog
header, `h` | v1 | reference to the document stream for a Bluesky run
`cat` | v2 | `Intake`-style databroker catalog (this is what we use now)
`run` | v2 | reference to a Bluesky run
`run.primary.read()` | v2 | returns an xarray object filled with the run's primary data stream

Reply:
I believe it is confusing to present the v1 terms.  The only places I use them now:

- `RE.subscribe(cat.v1.insert)`
- accessing the stream of documents directly: `for key, doc in h.documents(): ...`
- documentation that needs to be updated

Reply:
To get the run's metadata:  `run.metadata` returns a `dict`.  The keys are: `['start', 'stop', 'catalog_dir']`

Reply:
The difference between Broker and Intake style catalogs is seen by the user in how much work they must do to access the acquired data.  

The (original) Broker-style catalog focused on the ability to replay the documents in sequence as they were acquired.  As a consequence, the user had to understand how to step through these documents to build the data arrays for their measurement.

This was too tedious for most users.  The Intake-style catalog was an upgrade which focused on _using the data_ as it was organized into named streams,.

Another difference affects the ones who install and maintain the local catalog definitions.  The location and YAML format of the catalog definition file changed between Broker and Intake catalogs.  For the APS, I shifted all catalog definitions to Intake-style about a year ago.